plugin/storage/es/README.md

```diff
@@ -64,3 +64,26 @@ This script requires Docker to be running.
 Integration test framework for storage lie under `../integration`. 
 Add to `../integration/fixtures/traces/*.json` and `../integration/fixtures/queries.json` to add more
 trace cases.
+
+## Example of cleaner job configuration for Kubernetes
```

Comment:
We don't want to document it here. There is an operator which handles k8s deployments.

The kubernetes repository contains a similar command for spark dependencies. The repository is deprecated but perhaps we could add a command for index cleaner there.

https://github.com/jaegertracing/jaeger-kubernetes#service-dependencies

```
kubectl run jaeger-spark-dependencies --schedule="55 23 * * *" --env="STORAGE=elasticsearch" --env="ES_NODES=elasticsearch:9200" --env="ES_USERNAME=changeme" --env="ES_PASSWORD=changeme" --restart=Never --image=jaegertracing/spark-dependencies
```

Reply:
I think we could have an example here for reference, and add a notice saying that the recommended way to deploy Jaeger in Kubernetes is via the Operator.

Reply:
There are already 3 repositories with kubernetes related docs and manifest (jaeger-kubernetes, jaeger-openshift, jaeger-operator). We do not have any kubernetes docs in this main repo.

My preference would be not adding it here.

Reply:
so i may pull into https://github.com/jaegertracing/jaeger-kubernetes ? i am on minikube and aks, not openshift. we have tried jaeger-operator and offical es operator - these are too intrusive and do not fit.

Reply:
so modified page of cleaner in jaegertracing/jaeger could point to page about cleaner at jaegertracing/jaeger-kubernetes or at jaeger-openshift, which could suggest to use jaeger-operator .

Reply:
I would love to hear more about your experience with the operator. What made it a deal breaker?

Reply:
https://github.com/jaegertracing/jaeger-kubernetes seems like a better place to document it. There is already documentation for spark dependencies job. Feel free to submit a PR there.


+1 on hearing why you cannot use the operator. We are constantly improving it and feedback really helps us to drive better adoption of the project.